src/services/leaderboard/leaderboard.service.js

```diff
@@ -0,0 +1,26 @@
+// Initializes the `Leaderboard` service on path `/leaderboard`
```

Comment:
Ideally, the changes for #65 should be moved to a second PR in a new branch. This makes the reviewing process easier and also allows you to test each feature independently.